### PR TITLE
Recursion

### PR DESCRIPTION
**Pull request by VikasPandey121**
## Contributed in #1 With a Directory named recursion_Algorithms where all the algorithms implementing recursion can be added.

**Added Merge Sort**

### CODE REVIEW

**Review by chanmol1999 — CHANGES_REQUESTED** [review]
Move this inside algorithms directory

File: recursion_Algorithms/Merge-Sort.cpp [hunk 1]
```diff
@@ -0,0 +1,67 @@
+#include<iostream>
+using namespace std;
+
+void merge(int *a,int s,int e)
+{
```
Comment (Bhavnaharitsa) [inline recursion_Algorithms/Merge-Sort.cpp:5]: what is s and what is e.... mention

File: recursion_Algorithms/Merge-Sort.cpp [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    int a[100];
+    int n;
+    cin>>n;
+    for(int i=0;i<n;i++)
+    {
+        cin>>a[i];
+    }
+    mergeSort(a,0,n-1);
+    for(int i=0;i<n;i++)
+    {
```
Comment (Bhavnaharitsa) [inline recursion_Algorithms/Merge-Sort.cpp:63]: why nothing in cout?

File: recursion_Algorithms/Merge-Sort.cpp [hunk 3]
```diff
@@ -0,0 +1,67 @@
+#include<iostream>
```
Comment (Bhavnaharitsa) [inline recursion_Algorithms/Merge-Sort.cpp:1]: add brief description of the algorithm with sample inputs and outputs

**Comment by chanmol1999** [comment]
No Response going to close

**Comment by VikasPandey121** [comment]
I was busy with my end term exams. Now my exams are over and I can continue working on this PR.

**Comment by chanmol1999** [comment]
Ok Go ahead with this or you can create a new one.
But finish it by today.

**Pull request closed without merging**